src/utils/api/response.ts

```diff
@@ -29,6 +29,8 @@ const config = getConfigOrThrow();
 const retries: number = 10;
 const delay: number = 3000;
 const timeout: Millisecond = config.CHECKOUT_API_TIMEOUT as Millisecond;
+const decode = (transactionId: string): string => Buffer.from(transactionId, 'base64').toString('binary');
+
```

Comment:
The base64 string sent from transactions-service is calculated starting from the bytes of the UUID (high and low bytes) so here you have to build UUID structure again as shared [here](https://playcode.io/1195321)

Reply:
You are absolutely right

Reply:
@pietro-tota  What do you think now 😉